doc/architecture/performance/generator.rst

```diff
@@ -0,0 +1,70 @@
+Generator
+=========
+
+Overview
+--------
+
+This component is responsible for generating requests and storing them in a format 
+that could be submitted to the server. The user can declare the requests by leveraging
+the functions inside the library in ``CCF/tests/perf-system/generator/generator.py``.

```

Comment:
Could you make this and all other references to specific files into links

Reply:
If you look for instances of `ccf_repo` in the rst files, that'll show how we link to files in Github.